templates/python/.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,49 @@
+name: Pre-commit
```

Comment:
I think I would prefer something more general such as _CI_ for the name of this workflow (and filename)
that's the same we already use for both kpops and streams-explorer

or do you have an idea?

Reply:
_CI_ is good. I did notice _CI_ was used in the other projects, but I wasn't sure if it fits as the file looks a lot more specific than the _CI_s you mentioned. For example in streams-explorer's ci.yml,  _Publish_ is also a job.